src/mousemanager.cpp

```diff
@@ -186,20 +171,18 @@ void MouseManager::mouse_stop_drag() {
 }
 
 void MouseManager::handle_motion_event(Point2D newCursorPos) {
-    if (!draggingIsStillSafe()) {
-        mouse_stop_drag();
+    if (!dragHandler_) {
```

Comment:
Would it make sense, semantically, to call `mouse_is_dragging` here (and in similar situations)?

Reply:
No, it would not make sense in my opinion, because the check here is less semantically, but more technically: it ensures that `dragHandler_` is not null when dereferencing it in the next line:
```
dragHandler_->handle_motion_event(newCursorPos);
```